src/CoreWCF.Primitives/src/CoreWCF/Dispatcher/SecurityServiceDispatcher.cs

```diff
@@ -640,13 +639,13 @@ private async ValueTask<Message> ProcessInnerItemAsync(Message innerItem, TimeSp
             }
             if (securityException != null)
             {
-                SendFaultIfRequired(securityException, unverifiedMessage, timeoutHelper.RemainingTime());
+                await SendFaultIfRequired(securityException, unverifiedMessage, timeoutHelper.RemainingTime());
                 return null;
             }
             return innerItem;
         }
 
-        private void SendFaultIfRequired(Exception e, Message unverifiedMessage, TimeSpan timeout)
+        private async Task SendFaultIfRequired(Exception e, Message unverifiedMessage, TimeSpan timeout)
```

Comment:
Can you change the method names that are now async to have the suffix Async?

Reply:
Ah, good catch. WIll fix